README.md

```diff
@@ -1,2 +1,89 @@
 # github-release-orb
-Civis Analytics' CirleCI Orb repo.
+Civis Analytics' CircleCI Orb repo.
+
+## Usage
+
+Example Config:
+
+```
+version: 2.1
+
+orbs:
+  github-release: civisanalytics/github-release@x.y.z
+
+jobs:
+  release:
+    executor: github-release/github
+  steps:
+    - github-release/<command>
+```
+
+## Build
+In order to build the orb you can run the following command in the root directory of this repository.
+
+`circleci config pack src > orb.yml`
+
+Do not commit the created `orb.yml` file. This file is for publishing to CircleCI when finished.
+
+## Required Environment Variables
+In order for this orb to work, you must have three Environment Variables set:
+1. `GITHUB_TOKEN` - Your GitHub personal access token.
+2. `CIRCLE_PROJECT_USERNAME` - GitHub repository's user or organization. This should be set automatically.
+3. `CIRCLE_PROJECT_REPONAME` - GitHub repository name. This should be set automatically.
+
+## Commands
+
+### Publish
+Publish a new GitHub Release if the last version does not match the current version. This command is useful for running within a custom job. If no custom job is needed, we recommend using the built-in `publish-github-release` job provided by this orb.
+
```

Comment:
Can you give an example of when a custom job would be useful? At least just for my own sake. Would this allow you to automate finding the current-version, for example?

Reply:
maybe you want a custom job to first create a file to upload with the release that doesn't exist yet, kinda thinking along the lines of adding the JAR for the JDBCTool, then you use the created file-path in the command.

Reply:
i added to the examples a `custom-job`